bin/common.sh

```diff
@@ -38,11 +48,20 @@ function abort() {
 }
 trap abort SIGINT
 
-#######################################
+#####
 # https://github.com/google/styleguide/blob/gh-pages/shellguide.md#stdout-vs-stderr
-#######################################
+#####
 function err() {
-  echo "[$(date +'%Y-%m-%dT %T.%3N')] ERROR: $*" >&2
+  local tab=$'\t'
```

Comment:
I propose to extend this error message. I think it could really help with debugging (thanks @Dunky13 for the set-up). It will look like this e.g.:

```
otomi bats
Error: unknown command "r" for "yq"
Run 'yq --help' for usage.
Error: unknown command "r" for "yq"
Run 'yq --help' for usage.
bin/otomi: line 235: bats: command not found
---------------------------                        --------------------------- 
- Time                                             [2021-03-02T 08:27:28.456]
- Faulty script                                    bin/otomi
- Last function                                    drun
- Last arguments                                   $command
- Script message (if any)                          common.sh FAILED
```